README.md

```diff
@@ -335,6 +335,8 @@ Additionally, you can also locally trigger a deep link to make sure your code is
 
 ## Rich push notifications
 
+> :warning: **Rich push notification is supported from SDK version [2.2.0](https://github.com/klaviyo/klaviyo-swift-sdk/releases/tag/2.2.0)**
```

Comment:
What do you think about the following?

Rich push notifications are supported in SDK version [2.2.0] and higher (https://github.com/klaviyo/klaviyo-swift-sdk/releases/tag/2.2.0)

Reply:
I like it. Will update now.

Reply:
That sounds better to me.